snapshottest/module.py

```diff
@@ -225,14 +226,18 @@ def store(self, data):
         self.module[self.test_name] = data
 
     def assert_value_matches_snapshot(self, test_value, snapshot_value):
+        if self.ignored_keys is not None:
+            self.clear_ignored_keys(test_value)
+            self.clear_ignored_keys(snapshot_value)
         formatter = Formatter.get_formatter(test_value)
         formatter.assert_value_matches_snapshot(self, test_value, snapshot_value, Formatter())
 
     def assert_equals(self, value, snapshot):
         assert value == snapshot
 
-    def assert_match(self, value, name=''):
+    def assert_match(self, value, name='', ignored_keys=None):
```

Comment:
To me `ignore_keys` sounds better.
It's like "assert the match but ignore these keys" :)

Reply:
The difference is subtle but yes `ignore_keys` might be better.

I thought about this and the deciding factor for me `ignored_keys` vs `ignore_keys` is whether I think the "ignoring" had already happened or is going to happen.

One could argue that the ignoring happened when the user decided it's supposed to be ignored then `ignored_keys` feels right. On the other hand the actual clearing/ignoring of the data in the code is yet to happen so then `ignore_keys` is a better fit.